worker_test.go

```diff
@@ -1135,6 +1137,37 @@ func TestMatchIngressesToLoadbalancers(t *testing.T) {
 				require.Equal(t, 1, len(lb.ingresses["foo"]))
 			}
 		},
+	}, {
+		title: "load balancer with invalid subnets",
+		ingresses: []*kubernetes.Ingress{{
+			Name: "foo-ingress",
+			Hostnames: []string{
+				"foo.org",
+				"bar.org",
+			},
+			LoadBalancerType: awsAdapter.LoadBalancerTypeNetwork,
+			Shared:           true,
+		}},
+		lbs: []*loadBalancer{{
+			loadBalancerType: awsAdapter.LoadBalancerTypeNetwork,
+			ingresses:        make(map[string][]*kubernetes.Ingress),
+			stack:            &aws.Stack{Subnets: []string{"a", "b", "c"}},
+		}},
+		validate: func(t *testing.T, lbs []*loadBalancer) {
+			require.Equal(t, 3, len(lbs))
+			for _, lb := range lbs {
+				if lb.clusterLocal {
+					continue
+				}
+
+				if lb.stack != nil && equalSlices[string](lb.stack.Subnets, []string{"a", "b", "c"}) {
```

Comment:
I guess you also want to change this to `slices.Equal` as GH actions show `undefined: equalSlices`

Reply:
Fixed!